src/main/java/org/jitsi/videobridge/shim/ChannelShim.java

```diff
@@ -242,23 +242,13 @@ public void describe(ColibriConferenceIQ.ChannelCommon commonIq)
                 iq.addSource(bridgeSource);
             }
 
-            if (sources != null && !sources.isEmpty())
-            {
-                int[] ssrcs = new int[sources.size()];
-                int i = 0;
-                for (SourcePacketExtension source : sources)
-                {
-                    long ssrc = source.getSSRC();
-                    if (ssrc != -1)
-                    {
-                        ssrcs[i] = (int) source.getSSRC();
-                        i++;
-                    }
-                }
-                if (i < ssrcs.length)
-                {
-                    ssrcs = Arrays.copyOf(ssrcs, i);
-                }
+            if (sources != null) {
```

Comment:
Are these changes related to the debug features?

Reply:
they are not, consider it a [rider](https://en.wikipedia.org/wiki/Rider_(legislation))...